svf-llvm/lib/extapi.c

```diff
@@ -1193,3 +902,208 @@ void _ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEC1EPKcRKS3_(void **arg
 {
     *arg0 = arg1;
 }
+
+const unsigned short **__ctype_b_loc(void)
+{
+    return STATIC_OBJECT;
+}
+
+int **__ctype_tolower_loc(void)
+{
+    return STATIC_OBJECT;
+}
+
+int **__ctype_toupper_loc(void)
+{
+    return STATIC_OBJECT;
+}
+
+int *__errno_location(void)
+{
+    return STATIC_OBJECT;
+}
+
+int * __h_errno_location(void)
+{
+    return STATIC_OBJECT;
+}
+
+void* __res_state(void)
+{
+    return STATIC_OBJECT;
+}
+
+char *asctime(const void *timeptr)
+{
+    return STATIC_OBJECT;
+}
+
+char * bindtextdomain(const char * domainname, const char * dirname)
+{
+    return STATIC_OBJECT;
+}
+
+char * bind_textdomain_codeset(const char * domainname, const char * codeset)
+{
+    return STATIC_OBJECT;
+}
+
+char *ctermid(char *s)
+{
+    return STATIC_OBJECT;
+}
+
+char * dcgettext(const char * domainname, const char * msgid, int category)
+{
+    return STATIC_OBJECT;
+}
+
+char * dgettext(const char * domainname, const char * msgid)
+{
+    return STATIC_OBJECT;
+}
+
+char * dngettext(const char * domainname, const char * msgid, const char * msgid_plural, unsigned long int n)
+{
+    return STATIC_OBJECT;
+}
+
+void *fdopen(int fd, const char *mode)
+{
+    return STATIC_OBJECT;
```

Comment:
As far as I know, fdopen will (just like fopen & family) return a new object every time it's called. Should this be ALLOC_RET instead?